vanilla-framework/vanilla-dashboard-theme
# Tabs

## Done

Creates desktop and mobile view for tabs navigation

## QA

- Build the branch `./run`
- View pattern here http://0.0.0.0:8102/vanilla-dashboard-theme/examples/patterns/tabs/tabs/ and check against designs
- Designs https://github.com/ubuntudesign/vanilla-dashboard-theme-design/tree/master/Tabs

## Details

Fixes #14
